lib/bitpal/backends/flowee/protocol.ex

```diff
@@ -169,20 +169,52 @@ defmodule BitPal.Backend.Flowee.Protocol do
       [:offset | rest] -> [{44, true} | get_output(rest)]
       [:inputs | rest] -> [{46, true} | get_output(rest)]
       [:outputs | rest] -> [{49, true} | get_output(rest)]
+      [:outputScripts | rest] -> [{48, true} | get_output(rest)]
       [:outputAddrs | rest] -> [{50, true} | get_output(rest)]
+      [:outputHash | rest] -> [{51, true} | get_output(rest)]
       [:amounts | rest] -> [{47, true} | get_output(rest)]
     end
   end
 
+  # Get filters for the get operation.
+  defp get_filters(filters) do
+    case filters do
+      [] -> []
+      [{:address, hash} | rest] -> [{42, %Binary{data: hash}} | get_filters(rest)]
```

Comment:
This works of course, but in Elixir we typically use `Enum.map(filters, fn {:address, hash} -> ... end)` or similar